src/keymgmt.c

```diff
@@ -1473,7 +1477,7 @@ static int p11prov_ec_secbits(int bits)
     if (bits < 384) {
         return 128;
     }
-    if (bits < 512) {
+    if (bits < CURVE_521_BITS) {
```

Comment:
Is the change of the value (512 -> 521) intentional here or is it a typo? OpenSSL's default implementation uses 512 as the threshold as was the case before this change.

Reply:
Well given there is no 512 bit curve the distinction doesn't really exist.
But I can revert this change if you think this can be too confusing.

Reply:
I actually reverted this change, given it does not really belong in a coverity related fix anyway.